README.md

```diff
@@ -17,17 +17,28 @@ instance, includes a
 [Cockpit Module](https://github.com/osbuild/cockpit-composer) or using the
 [composer-cli](https://weldr.io/lorax/composer-cli.html) command-line tool.
 
-### Project
+## Project
 
  * **Website**: <https://www.osbuild.org>
  * **Bug Tracker**: <https://github.com/osbuild/osbuild-composer/issues>
- * **matrix**: https://matrix.to/#/#image-builder:fedoraproject.org
-*  **Mailing List**: image-builder@redhat.com
+ * **Matrix**: https://matrix.to/#/#image-builder:fedoraproject.org
+ * **Mailing List**: image-builder@redhat.com
  * **Changelog**: <https://github.com/osbuild/osbuild-composer/releases>
 
-#### Contributing
+### Principles
 
-Please refer to the [developer guide](https://www.osbuild.org/guides/developer-guide/index.html) to learn about our workflow, code style and more.
+1. OSBuild Composer shall only allow users to do what generally makes sense.
+2. Blueprints are the policy layer where we decide what to expose to end users.
+3. If a blueprint can be built, it should also boot.
```

Comment:
I'm still not sure about this one. It's so easy to break an image using a custom RPM/file.

Reply:
Right. I mean it's a principle and as such aspirational. What it means to me is: we don't hand out footguns, but it's  not unbreakable.

Reply:
I think the word "should" carries the right amount of weight for this (and is probably correct in the RFC meaning of the word too).

Reply:
@ondrejbudai As no further concerns have been raised - do you have an alternative suggestion or are you good with this as is for now?